src/pubsub-message.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+const isNode = require('detect-node')
+const PubsubMessage = require('../../src/pubsub-message') // eslint-disable-line no-unused-vars
```

Comment:
The tests have a `before` filter, which calls `common.setup`, so you can pass more things through there. You could either add the functionality to the `factory` or as a third parameter.